onboarding_project/templates/admin/squad_pantry_app/order/change_form.html

```diff
@@ -0,0 +1,12 @@
+{% extends "admin/change_form.html" %}
+{% load i18n admin_urls %}
+{% block object-tools-items %}
+    <li>
```

Comment:
Instead of re-defining the original `history` button, do a `{{ block.super }}` to inherit from the original change form

Reply:
Done!